applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -1034,24 +1030,65 @@ static int cmd_hfclkaudio_drift_comp_disable(const struct shell *shell, size_t a
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	ctrl_blk.drift_comp.hfclkaudio_comp_enabled = false;
+	if (ctrl_blk.pres_comp.enabled) {
+		shell_print(shell, "Pres comp must be disabled to disable drift comp");
+	} else {
+		ctrl_blk.drift_comp.enabled = false;
+		drift_comp_state_set(DRIFT_STATE_INIT);
+
+		shell_print(shell, "Audio PLL drift compensation disabled");
+	}
+
+	return 0;
+}
+
+static int cmd_audio_pres_comp_enable(const struct shell *shell, size_t argc, const char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	if (ctrl_blk.drift_comp.enabled) {
+		ctrl_blk.pres_comp.enabled = true;
+
+		shell_print(shell, "Audio PLL presentation compensation enabled");
+	} else {
+		shell_print(shell, "Drift comp must be enabled to enable pres comp");
+	}
+
+	return 0;
+}
+
+static int cmd_audio_pres_comp_disable(const struct shell *shell, size_t argc, const char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	ctrl_blk.pres_comp.enabled = false;
+	pres_comp_state_set(PRES_STATE_INIT);
 
-	shell_print(shell, "Audio PLL drift compensation disabled");
+	shell_print(shell, "Audio PLL presentation compensation disabled");
 
 	return 0;
 }
 
-SHELL_STATIC_SUBCMD_SET_CREATE(test_cmd,
-			       SHELL_COND_CMD(CONFIG_SHELL, nrf_tone_start, NULL,
-					      "Start local tone from nRF5340.", cmd_i2s_tone_play),
-			       SHELL_COND_CMD(CONFIG_SHELL, nrf_tone_stop, NULL,
-					      "Stop local tone from nRF5340.", cmd_i2s_tone_stop),
-			       SHELL_COND_CMD(CONFIG_SHELL, pll_comp_enable, NULL,
-					      "Enable audio PLL auto drift compensation (default).",
-					      cmd_hfclkaudio_drift_comp_enable),
-			       SHELL_COND_CMD(CONFIG_SHELL, pll_comp_disable, NULL,
-					      "Disable audio PLL auto drift compensation",
-					      cmd_hfclkaudio_drift_comp_disable),
-			       SHELL_SUBCMD_SET_END);
+SHELL_STATIC_SUBCMD_SET_CREATE(
```

Comment:
All help texts should either have period at the end or no period. You can choose, since it is not consistent in the application, hehe :)